internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestAdminTool.java

```diff
@@ -0,0 +1,170 @@
+    Assert.assertFalse(newStoreResponse.isError(), "Store creation failed with error: " + newStoreResponse.getError());
+    // Make sure the store exists in Child fabric
+    TestUtils.waitForNonDeterministicAssertion(10, TimeUnit.SECONDS, () -> {
+      StoreResponse storeResponse = childControllerClient.getStore(testStoreName);
+      Assert.assertFalse(storeResponse.isError(), "Store lookup failed with error: " + storeResponse.getError());
+      Assert.assertTrue(storeResponse.getStore() != null, "Store response shouldn't be null");
+    });
+
+    // Create a new version and write some data
+    VersionCreationResponse versionCreationResponse = parentControllerClient.requestTopicForWrites(
+        testStoreName,
+        10000,
+        Version.PushType.BATCH,
+        "test_push_id",
+        true,
+        true,
+        false,
+        Optional.empty(),
+        Optional.empty(),
+        Optional.empty(),
+        false,
+        -1);
+    Assert.assertFalse(
+        versionCreationResponse.isError(),
+        "Failed to create a new version, error: " + versionCreationResponse.getError());
+    testStoreVersionTopicName = versionCreationResponse.getKafkaTopic();
+    // Write some data to the new version
+    Properties props = new Properties();
+    props.setProperty(KAFKA_BOOTSTRAP_SERVERS, versionCreationResponse.getKafkaBootstrapServers());
+    VeniceWriterFactory veniceWriterFactory = new VeniceWriterFactory(props);
+    VeniceWriter veniceWriter = veniceWriterFactory.createVeniceWriter(
+        new VeniceWriterOptions.Builder(testStoreVersionTopicName)
+            .setKeySerializer(new VeniceAvroKafkaSerializer(schema))
+            .setValueSerializer(new VeniceAvroKafkaSerializer(schema))
+            .build());
+    veniceWriter.put(testKey, testValue, 1);
+    veniceWriter.broadcastEndOfPush(Collections.emptyMap());
+    veniceWriter.flush();
+    veniceWriter.close();
+
+    TestUtils.waitForNonDeterministicAssertion(30, TimeUnit.SECONDS, () -> {
+      StoreResponse storeResponse = childControllerClient.getStore(testStoreName);
+      Assert.assertFalse(storeResponse.isError(), "Store lookup failed with error: " + storeResponse.getError());
+      Assert.assertEquals(storeResponse.getStore().getCurrentVersion(), 1);
+    });
+
+  }
+
+  @AfterClass
+  public void tearDown() {
+    Utils.closeQuietlyWithErrorLogged(twoLayerMultiRegionMultiClusterWrapper);
+    Utils.closeQuietlyWithErrorLogged(parentControllerClient);
+    Utils.closeQuietlyWithErrorLogged(childControllerClient);
+  }
+
+  @Test
+  public void testDumpKafkaTopic() throws Exception {
+    String[] args = { "--dump-kafka-topic", "--kafka-bootstrap-servers", childKafkaAddress, // Version topic only exists
+        // in child Kafka clsuter
+        "--kafka-topic-name", testStoreVersionTopicName, "--kafka-topic-partition", "0", "--url", childControllerUrl,
+        "--cluster", clusterName, "--log-metedata", "true" };
+    AdminTool.main(args);
```

Comment:
Don't we have to verify anything? If no exception is considered good, we can catch any exception and fail the test in the catch block.